### DISCUSSION TITLE
Can it adapt to Dart formaldehyde sensor?

### DISCUSSION THREAD

**Discussion by wisonlau** [arendst/Tasmota#20836] [Ideas]
![image](https://github.com/arendst/Tasmota/assets/26670866/919dd083-35f6-4b92-964f-fce2105fe246)

I want to use tasmota to get formaldehyde data, but I can't find it in the support list. Can you support it in the future?

**Comment by sfromis** [comment]
Sure, if someone creates a driver; this is typically done by someone with a need sitting down to do the work, and submit to the project. Open Source, you know.....
